build_backend.py

```
@@ -25,6 +27,360 @@
+
+
+def _nixl_buildable() -> tuple[bool, str]:
+    """Probe for hard NIXL-EP build-time deps. Returns (ok, reason_if_not)."""
+    if not shutil.which("meson"):
+        return False, "meson not on PATH (apt install meson)"
+    if not shutil.which("ninja"):
+        return False, "ninja not on PATH (apt install ninja-build)"
+    pkgconfig = shutil.which("pkg-config")
+    if not pkgconfig:
+        return False, "pkg-config not on PATH (apt install pkg-config)"
+    r = subprocess.run([pkgconfig, "--exists", "ucx"], capture_output=True)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)